R/calculate_study_day.R

```diff
@@ -0,0 +1,142 @@
+#' `calculate_study_day` performs study day calculation
+#' @description
+#' This function takes the an input data.frame and a reference data.frame (which
+#' is DM domain in most cases), and calculate the study day from reference date
+#' and target date. In case of unexpected conditions like reference date is not
+#' unique for each patient, or reference and input dates are not actual dates,
+#' NA will be returned for those records.
+#'
+#' @md
+#' @param sdtm_in input data.frame that contains the target date.
```

Comment:
```suggestion
#' @param sdtm_in Input data frame that contains the target date.
```
Please use capitalized "I" for the first letter of a parameter. See reference here: https://style.tidyverse.org/documentation.html#documenting-parameters.